plugin/src/plugin.ts

```diff
@@ -27,7 +28,12 @@ module.exports = function (): PluginItem {
     visitor: {
       CallExpression: {
         enter(path: NodePath<CallExpression>, state: ReanimatedPluginPass) {
-          runWithTaggedExceptions(() => processForCalleesWorklets(path, state));
+          runWithTaggedExceptions(() => {
+            processForCalleesWorklets(path, state);
+            if (state.opts.substituteWebPlatformChecks) {
+              substituteWebCallExpression(path);
+            }
+          });
```

Comment:
We have to keep in mind this won't transform `shouldBeUseWeb` or `isWeb` inside worklets. While it's not a valid case to call them there, we should at least have it included in tests.